LanguageMaster.xml

```diff
@@ -1421,8 +1421,10 @@ can edit in same subtitle file (collaboration)</Information>
     <NewLine>New line (\r\n)</NewLine>
     <NewLineShort>New line (\n)</NewLineShort>
     <AnyDigit>Any digit (\d)</AnyDigit>
+    <NonDigit>Non digit (\D)</NonDigit>
     <AnyCharacter>Any character (.)</AnyCharacter>
     <AnyWhitespace>Any whitespace (\s)</AnyWhitespace>
+    <NonSpaceCharacter>Non space character (\S)</NonSpaceCharacter>
```

Comment:
English is not my native language, but "non-digit character" / "non-whitespace character" looks good to me - like from http://www.w3schools.com/jsref/jsref_regexp_digit_non.asp